sql_basics/sql_basics.md

```diff
@@ -589,6 +604,21 @@ You can also explore using `SELECT DISTINCT` on more than one field.  The code b
 
 Then, if you're up for a challenge, change the code to find out what unique combinations there are of race and ethnicity!
 
+
+<details>
```

Comment:
I suggest moving this hint to after the code block, so that the page goes:

1. instructions
2. code block
3. instructions for extra challenge
4. hint for extra challenge